cogs/commands.py

```diff
@@ -268,6 +268,16 @@ async def scoreboard(self, ctx):
             table.append((str(user), user.messages_sent))
 
         await ctx.send(f'>>> ```prolog\n{tabulate(table, headers=("User", "Messages",), tablefmt="fancy_grid")}\n```')
+                       
+                       
+    @commands.command(aliases = ['emj'])
+    async def emoji(self, ctx, *, name):
+        """Emoji"""
```

Comment:
This will appear in the help so change it to something like "Send an emoji, useful for those who don't have nitro but want to send animated emojis."